docs/release_notes/release_notes.md

```diff
@@ -9,67 +9,33 @@ has_children: true
+    * Added raw filter macro for "Network Compromise - Basic Scanning" alert.
 
-* ### Enhancements
-    * Enhanced MultiSelect functionality
-        * MultiSelect input will automatically select/unselect All option based on the user selection in all the dashboards.
-    
-    * Support for email messagetrace event using Splunk Add-on for Office 365. 
-        * Updated Email related alerts to support email messagetrace event collected using Splunk Add-on for Office 365..
+    * Enhanced "Email - Hourly Increase In Emails Over Baseline" alert to show domain wise recipient information.
 
-    * Reduced severity level for the already blocked events for Palo High Threat alert.
+    * Improved "Linux - User Added/Updated/Deleted" and "Linux - Group Added/Updated/Deleted" alerts to show exact changes detail.
+        * Deprecated "Linux - Change in Sudo Access of Local Linux Account" alert.
 
-    * Windows - Windows Process Tampering Detected alert
-        * Added internal filter macro to reduce the false positives.
-
-    * Added ApplicationId field in the all O365 Login related alerts.
-
-    * Improved "O365 - Login From Unknown User" alert to consider both Unknown and "Not Available" user values.
-
-    * Improved "O365 Daily Login Failure" alert readability.
-
-    * Excluded TriggerBrowserCapabilitiesInterrupt error events from "O365 - Daily Login Failure" alert to reduce false positives.
-        * More information for the error: https://login.microsoftonline.com/error?code=501314
-    
-    * Improved severity logic for "Email - Hourly Increase In Emails Over Baseline" alert to reduce false positives.
+    * Updated threshold value for the "Network Compromise - Basic Scanning" alert.
 
 * ### Bug Fixes
-    * Fixed minor issue in the DNS Tracker dashboard for zero event.
-
-    * Fixed windows decommissioned host reappearing issue.
-
-    * Fixed "Windows Defender RealTime Protection Disabled or Failed" alert search query.
-
-    * Fixed severity inconsistency between alert and overview dashboard for antivirus service stop alerts.
-
-    * Fixed Cyences logo issue.
-
+    * Fixed fetching first 50 records multiple times issue for sophos endpoint custom command.
 
-## Upgrade Guide from 3.1.0 to 4.0.0
+    * Fixed the permission issue to allow normal user to use notable event assignment functionality.
 
-* ### Cyences Settings
-    * Go through the [Cyences App Configuration]({{ site.baseurl }}/install_configure/configuration/#products-setup-data-source-macros) and make required changes
 
-    * The "Microsoft 365 Defender Add-on for Splunk" is deprecated. Use "Splunk Add-on for Microsoft Security" addon for O365 Defender ATP data collection. 
+## Upgrade Guide from 4.0.0 to 4.1.0
+    * New Cyences_Authentication Data model is added. User should accelerate Cyences_Authentication with minimum 1 month period to improve search performance.
```

Comment:
<img width="816" alt="image" src="https://github.com/CrossRealms/Splunk-Cyences-App-for-Splunk/assets/20964366/81887b35-d153-4af3-a7e7-52b8fdb3a8d0">